server/views/applications/pages/oasys-import/_oasys-screen.njk

```diff
@@ -17,7 +17,8 @@
       {% if oasysDisabled or page.oasysSuccess == false %}
         <h1 class="govuk-heading-l">Provide risk information</h1>
       {% else %}
-        <h1 class="govuk-heading-l">{{page.title}}</h1>
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-2">{{page.title}}</h1>
+        <p>{{page.riskTaskStep}}</p>
```

Comment:
If `riskTaskStep` was an integer then in the template we could have:
`<p>Risk Information (part {{page.riskTaskStep}} of 5)</p>`
so there would be less places to update if we ever need to change the copy (EG if a page was added/removed).

Reply:
Thanks for making the edit I suggested in the OasysPage classes but now `page.riskTaskStep` is an integer we have to update this line to be `<p>Risk Information (part {{page.riskTaskStep}} of 5)</p>`

Reply:
yah sorry the commit push failed because of token expired and pushed again please check @richpjames